go-backend/migrations/1587467112_create_organisation_table.up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE organizations (
+  id serial PRIMARY KEY,
+	name varchar(50),
+	contact_email varchar(50),
+	domain_name varchar(45),
+	subscription_status integer,
+	subscription_valid_upto BIGINT,
+	hi5_limit integer,
+	hi5_quota_renewal_frequency varchar(50),
+	timezone varchar(100),
+	created_by integer,
+	created_on BIGINT DEFAULT extract(epoch from now()),
```

Comment:
not sure whats correct ?
but i think we agreed on created_at, updated_at 

check with @sunil4sonawane

Reply:
Please check with updated schema on Wiki. We have removed these fields already.